src/main/scala/pipelines/text/AmazonReviewsPipeline.scala

```diff
@@ -71,11 +72,11 @@ object AmazonReviewsPipeline extends Logging {
     val conf = new SparkConf().setAppName(appName)
     conf.setIfMissing("spark.master", "local[2]") // This is a fallback if things aren't set via spark submit.
 
-    val sc = new SparkContext(conf)
+    val spark = SparkSession.builder.config(conf).getOrCreate()
```

Comment:
Yeah, I dont think its deprecated per-se but all Dataset, DataFrame operations need SparkSession and I think many users are switching to that. So #291 would be good to have.